app/helpers/application_helper.rb

```diff
@@ -147,7 +147,7 @@ def concurrent_events(event)
   end
 
   def speaker_links(event)
-    event.speakers.map{ |speaker| link_to speaker.name, admin_user_path(speaker) }.join(', ').html_safe
+    event.speakers.map{ |speaker| link_to speaker.name, admin_user_path(speaker) }.join(', ').safe_join
```

Comment:
This is unrelated to the commit, I guess Rubocop was failing and that's why you fixed it, but please add it in a different commit. :wink: